monailabel/scribbles/utils.py

```diff
@@ -126,10 +126,10 @@ def get_values(_data, _seed, _label):
         return _values
 
     values = get_values(image, scrib, scribbles_bg_label)
-    bg_hist, bg_bin_edges = np.histogram(values, bins=512, range=(0, 1), density=True)
+    bg_hist, bg_bin_edges = np.histogram(values, bins=32, range=(0, 1), density=True)
```

Comment:
bins i would prefer to keep it in function args with default=32 in this case.. so that while using depending on the use-case, caller can decide

Reply:
Sure, I will update to have this in the code.